### ISSUE TITLE
Edit Products M3 - Product Categories Step 1: display product categories in Product detail screen

### ISSUE THREAD

**Issue by anitaa1990** [woocommerce/woocommerce-android#2185]
As part of m3 release, we should provide an option to display the categories of a product in the product detail screen.

<img src="https://user-images.githubusercontent.com/22608780/78359227-433eea00-75d2-11ea-9d1e-455883dd7a02.png" width="300" />

**Comment by mhaqs** [comment]
Hey @anitaa1990. I've started working on this one. I do have one question regarding the display of categories on the product page.

- What do we show to the user, if no categories are assigned?
- How many lines of categories do we show? If it's a single line, do we want to truncate with a `...` after a certain length?

@Garance91540 would you be able to help on this front?

**Comment by anitaa1990** [comment]
> What do we show to the user, if no categories are assigned?

For all the other sections in the Product Detail screen, if there is no info to be displayed, we just display: "Add Pricing" or "Add Inventory". Perhaps we could just display "Add Categories".

> What do we show to the user, if no categories are assigned?

I think we can wrap the categories to be displayed upto 5 lines? I don't see a list of categories for a product going beyond that. But would be good to get @Garance91540's thoughts on this first.

**Comment by Garance91540** [comment]
@anitaa1990 @mhaqs I wanted to clarify something: Category is not a section that appears by default on the product page. Users have to select the bottom sheet > "Categories" to add it. @anitaa1990 do you know when we'll support the bottom sheet?

Once users have added "Categories", we can show a placeholder text of "Add Category" as @anitaa1990 suggested. 

> I think we can wrap the categories to be displayed upto 5 lines

Sounds good to me!

**Comment by anitaa1990** [comment]
> Category is not a section that appears by default on the product page. Users have to select the bottom sheet > "Categories" to add it. @anitaa1990 do you know when we'll support the bottom sheet?

I believe we added bottom sheet for sorting and we can reuse that in this case. I noticed that the bottom sheet is needed for Product short description and categories and tags as well. So we might need to make changes to other parts of the product detail screen not related to categories. I will open another issue for this to be handled separately once the categories feature is completed 👍
